src/internal/functions/Get-ReadmeContent.ps1

```diff
@@ -234,6 +236,27 @@ For specific information on troubleshooting a generated module, check the [troub
 
 Currently, because of the way Puppet caches files on agents, use of the legacy [``puppetlabs-dsc``]($LegacyDscForgePage) module is **not** compatible with this or any auto-generated DSC module.
 Inclusion of both will lead to pluginsync conflicts.
+
+### Module Installation
+
+If you're using this module with Puppet Enterprise and Code Manager, everything should "just work" - no errors or issues acquiring and deploying this or any Puppetized DSC module to nodes.
+
+Unfortunately, due a [bug in minitar], both [``r10k``]($r10kInstallationIssue) and [serverless Puppet (via ``puppet module install``)]($PuppetModuleInstallIssue) methods of installing modules with long path names will fail.
```

Comment:
Should there be a link here attached to `[bug in minitar]`?

Reply:
good catch! There's unfortunately not an actual bug filed, so I'll expand and update.